features/storage/csi_resize.feature

```diff
@@ -0,0 +1,45 @@
+Feature: CSI Resizing related feature
+  # @author chaoyang@redhat.com
+  @admin
+  Scenario Outline: Resize online volume from 1Gi to 2Gi
+    Given I have a project
+    And admin clones storage class "sc-<%= project.name %>" from "<sc_name>" with:
+      | ["metadata"]["name"] | sc-<%= project.name %> |
+    Then the step should succeed
```

Comment:
We do not overwrite any attributes for the storage class ?   Sound like we do not need the clone storage class there then.

Reply:
@liangxia Thanks for the comments. Have updated.